src/components/Navbar.js

```diff
@@ -1,41 +1,33 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import '../App.css';
 
-const Navbar = () => {
-  const links = [
-    {
-      id: 1,
-      path: '/books',
-      text: 'Books',
-    },
-    {
-      id: 2,
-      path: '/categories',
-      text: 'Categories',
-    },
-  ];
-  return (
-    <header>
-      <nav>
-        <div className="menu">
-          <h1>Bookstore CMS</h1>
-          <ul>
-            {links.map((link) => (
-              <li key={link.id}>
-                <Link
-                  to={link.path}
-                  className="nav-link"
-                  exact="true"
-                >
-                  {link.text}
-                </Link>
-              </li>
-            ))}
-          </ul>
-        </div>
-      </nav>
-    </header>
-  );
-};
+const Navbar = () => (
+  <nav className="navbar navbar-expand-lg navbar-light bg-light m-4">
+    <div className="container-fluid">
+      <Link className="navbar-brand  fw-bold" to="/">
+        <h4 className="rangi"> Bookstore CMS </h4>
+      </Link>
+      <div className="collapse navbar-collapse" id="navbarText">
+        <ul className="navbar-nav me-auto mb-2 mb-lg-0">
+          <li className="nav-item m-3 text-black">
+            <Link className="nav-link" to="/">
+              Books
+            </Link>
+          </li>
+          <li className="nav-item m-3 text-black">
+            <Link className="nav-link" to="/categories">
+              Categories
+            </Link>
+          </li>
+        </ul>
+      </div>
+      <div className="d-flex">
+        <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="currentColor" className="bi bi-person-fill rangi" viewBox="0 0 16 16">
+          <path d="M3 14s-1 0-1-1 1-4 6-4 6 3 6 4-1 1-1 1H3zm5-6a3 3 0 1 0 0-6 3 3 0 0 0 0 6z" />
+        </svg>
+      </div>
+    </div>
+  </nav>
+);
```

Comment:
- [ ] The NavBar should match the rest of the layout and it should have a different background color, Kindly update your head to match the Zeplin design.
![image](https://user-images.githubusercontent.com/33767951/141651125-b6114574-1a58-40be-a018-e4752e6d0908.png)
- [ ] Please update the `Navbar` background color to match the Zeplin design.